cmd/dfget/app/root.go

```diff
@@ -0,0 +1,207 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	cfg "github.com/dragonflyoss/Dragonfly/dfget/config"
+	"github.com/dragonflyoss/Dragonfly/dfget/core"
+	"github.com/dragonflyoss/Dragonfly/dfget/errors"
+	"github.com/dragonflyoss/Dragonfly/dfget/util"
+	"github.com/sirupsen/logrus"
```

Comment:
Add a blank here?
 We suggest split the imports into three group with a blank:
1. the packages belong to go standard library
2. the packages belong to our project 
3. the packages belong to Third-party

Reply:
Sure. I firstly use goland, it cannot help me to format import. I will revise it quickly.